sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5767,6 +5899,33 @@ bool _pi_queue::useCopyEngine(bool PreferCopyEngine) const {
          (!isInOrderQueue() || UseCopyEngineForInOrderQueue);
 }
 
+pi_result _pi_queue::waitOnAllImmCmdLists() {
+  if (!Healthy)
```

Comment:
I think I might have missed this. But what does 'Healthy' mean?

Reply:
A failed submit to a Queue results in the Queue being marked not Healthy.